modules/core/src/v2/coins/cspr.ts

```diff
@@ -31,10 +33,29 @@ export interface TransactionPrebuild extends BaseTransactionPrebuild {
   txJson: string;
 }
 
+export interface TransactionFee {
+  gasLimit: string;
+  gasPrice: string;
+}
+
+export interface ExplainTransactionOptions {
+  txHex?: string;
+  halfSigned?: {
+    txHex: string;
+  };
+  fee: TransactionFee;
+}
+
 interface SupplementGenerateWalletOptions {
   rootPrivateKey?: string;
 }
 
+interface TransactionOutput {
+  address: string;
+  amount: string;
+  coin: string;
```

Comment:
How come each output has a coin? Is it possible to send a tx which spends more than one type of coin?

Reply:
the coin will be used when we add support for token transactions